includes/class-sensei-lesson.php

```diff
@@ -3956,8 +4027,16 @@ public static function get_lesson_prerequisite_id( $current_lesson_id ) {
 
 		}
 
+		/**
+		 * Filter the lesson prerequisite.
+		 *
+		 * @since 1.0
+		 * @hook sensei_lesson_prerequisite
+		 *
+		 * @param {string|bool} $prerequisite_lesson_id Prerequisite lesson ID.
```

Comment:
It would be good to document the case when this is false.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/3565/commits/c44c3e4d1ef9ddab1b619f13de26ce24e330bb32.